perception/mil_vision/ros_tools/colorspaces.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+from __future__ import division, print_function
```

Comment:
I feel like that's not true, the transition will be likely automated right? Like a `flake_8` type thing we run over the whole repo. This just causes confusion in the mean time.